rollup/ccc/async_checker.go

```diff
@@ -94,8 +99,10 @@ func (c *AsyncChecker) Wait() {
 // Check spawns an async CCC verification task.
 func (c *AsyncChecker) Check(block *types.Block) error {
 	if block.NumberU64() > c.currentHead.Number.Uint64()+1 {
-		log.Error("non continuous chain observed in AsyncChecker", "prev", c.currentHead, "got", block.Header())
-	} else if block.ParentHash() != c.currentHead.Hash() {
+		log.Warn("non continuous chain observed in AsyncChecker", "prev", c.currentHead, "got", block.Header())
+	}
+
+	if block.ParentHash() != c.currentHead.Hash() {
```

Comment:
any triggering example of the follower `nil` consumption block here? e.g. it receives some future blocks first or?

Reply:
I see. if the future blocks come, and then the previous blocks are reverted due to rust ccc failure, the `forkCtx` may be canceled. triggering early return of `checkerTask`.